scout/load/hgnc_gene.py

```diff
@@ -78,20 +95,36 @@ def load_hgnc_genes(
             genemap_lines=genemap_lines,
         )
 
-    non_existing = 0
+    without_coords = 0
     nr_genes = len(genes)
     LOG.info(f"Building info for {nr_genes} genes")
+
+    cytoband_coords: Dict[str, dict] = adapter.cytoband_to_coordinates(build=build)
+
     with progressbar(genes.values(), label="Building genes", length=nr_genes) as bar:
         for gene_data in bar:
+            set_gene_coordinates(gene_data=gene_data, cytoband_coords=cytoband_coords)
```

Comment:
Hm, if I read this correctly you are always overwriting the more exact gene coords with the cytoband coords? That doesn't seem quite right! 😊 Perhaps a check for existing coords, or set only if the chrome, start stop dont already exist? You do check for ensembl gene id, which may or may not be enough there - maybe just rename the function slightly then?

Reply:
No no, it's not overwriting coords for genes that already have them, see line 23 of that file:

<img width="892" alt="image" src="https://github.com/user-attachments/assets/dd51998b-0ccb-40ff-a54d-9bb509e0cf89">

I'll rename the function as you suggested!